### ISSUE TITLE
The button of 'wrap table content' is ineffective on query log page

### ISSUE THREAD

**Issue by quinniup** [openobserve/openobserve#4328]
### Which OpenObserve functionalities are the source of the bug?

log search

### Is this a regression?

Yes

### Description

https://github.com/openobserve/openobserve/issues/3919

### Please provide the version you discovered this bug in (check about page for version information)

```true
Version：v0.11.0-rc1
Commit Hash：f0cd9fd3b10214b5d8df26f4e265c5302ee5157f
Build Date：2024-08-19T04:09:46Z
```

**Comment by hengfeiyang** [comment]
@omkarK06, i know we are refactor table for logs page, is it done? then we can fix the `wrap` issue. the user waiting for this fix for a couple month.

**Comment by hengfeiyang** [comment]
We fixed it in the latest release `v0.11.0`

**Comment by prabhatsharma** [comment]
fixed by https://github.com/openobserve/openobserve/pull/4097
